src/test/java/org/jenkinsci/plugins/gitclient/GitClientTest.java

```diff
@@ -257,6 +257,36 @@ private String randomEmail(String name) {
         return name.replaceAll(" ", ".") + "@middle.earth";
     }
 
+    @Test
+    @Issue("JENKINS-297799")
+    public void testChangelongVeryLong() throws Exception {
```

Comment:
```suggestion
    /* Changelog first line was truncated on word boundary prior to 
     * 72 characters with git client plugin 2.0+ with CLI git.
     * Was not truncated by git client plugin earlier releases using CLI git.
     * Was not truncated by git client plugin and JGit.
     * Rely on caller to perform truncation of first line if desired.
     * Matching change will be included in git plugin 4.0.0
     * to retain existing truncation behavior.
     */
    public void testChangelogVeryLong() throws Exception {
```

Reply:
I'd rather go for this:
```
    /**
     *  Changelog was formatted on word boundary prior to
     * 72 characters with git client plugin 2.0+ when using CLI git.
     * Was not truncated by git client plugin using JGit (And Apache version).
     * Rely on caller to truncate first line if desired.
     * Matching change will be included in git plugin 4.0.0
     * to retain existing truncation behavior.
     */

```

Reply:
I believe your proposed comment is incorrect, or at least imprecise when it says:

> Changelog was formatted on word boundary prior to 72 characters with git client plugin 2.0+ when using CLI git.

The **first line** of the changelog was **truncated** on a word boundary prior to 72 characters.  The rest of the changelog was not truncated, only the first line.

Reply:
The whole message was formatted, not only the first line.
This is the outcome when running the tests (and adding a s.out)


>     Uno Dos Tres Cuatro Cinco Lorem ipsum dolor sit amet, consectetur
>     adipiscing elit. Ut posuere tellus eu efficitur tristique. In iaculis
>     neque in dolor vulputatesollicitudin eget a quam. Donec finibus sapien
>     quis lectus euismod facilisis. Integermassa purus, scelerisque id
>     iaculis ut, blandit vitae velit. Pellentesque lobortisaliquet felis, vel
>     laoreet ipsum tincidunt at. Mauris tellus est, cursus vitae exeget,
>     venenatis auctor eros. Sed sagittis porta odio. Donec ut interdum massa.
>     Aliquamsagittis, mi sit amet sollicitudin elementum, velit quam eleifend
>     nisl, in rhoncusfelis nibh eu nibh. Class aptent taciti sociosqu ad
>     litora torquent per conubia nostra, per inceptos himenaeos. seis
> 
>     asfasfasfasf

whereas with the new formatting is: 

>

>     Uno Dos Tres Cuatro Cinco Lorem ipsum dolor sit amet, consectetur adipiscing elit. Ut posuere tellus eu efficitur tristique. In iaculis neque in dolor vulputatesollicitudin eget a quam. Donec finibus sapien quis lectus euismod facilisis. Integermassa purus, scelerisque id iaculis ut, blandit vitae velit. Pellentesque lobortisaliquet felis, vel laoreet ipsum tincidunt at. Mauris tellus est, cursus vitae exeget, venenatis auctor eros. Sed sagittis porta odio. Donec ut interdum massa. Aliquamsagittis, mi sit amet sollicitudin elementum, velit quam eleifend nisl, in rhoncusfelis nibh eu nibh. Class aptent taciti sociosqu ad litora torquent per conubia nostra, per inceptos himenaeos.
>     seis
>     
>     asfasfasfasf